chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -41,49 +41,47 @@
 default[:python][:installed] = "#{node[:python][:scripts]}\\#{node[:python][:pywin32register]}"
 
-default[:openstack][:location] = 'C:\OpenStack'
+default[:openstack][:location] = "C:\\OpenStack"
+default[:openstack][:src] = "#{node[:openstack][:location]}\\src"
+default[:openstack][:instances] = "#{node[:openstack][:location]}\\Instances"
+default[:openstack][:config] = "#{node[:openstack][:location]}\\etc"
+default[:openstack][:bin] = "#{node[:openstack][:location]}\\bin"
+default[:openstack][:log] = "#{node[:openstack][:location]}\\log"
+
+default[:openstack][:tarball_branch] = "stable-liberty"
 
 default[:openstack][:nova][:name] = "nova"
-default[:openstack][:nova][:version] = "2014.2.1"
-default[:openstack][:nova][:file] = "nova-2014.2.1.tar.gz"
-default[:openstack][:nova][:target] = "#{node[:openstack][:location]}"
-default[:openstack][:nova][:installed] = "#{node[:python][:path]}\\Scripts\\nova-compute.exe"
+default[:openstack][:nova][:binary] = "#{node[:python][:scripts]}\\nova-compute.exe"
+default[:openstack][:nova][:config] = "#{node[:openstack][:config]}\\nova"
 
 default[:openstack][:neutron][:name] = "neutron"
-default[:openstack][:neutron][:version] = "2014.2.1"
-default[:openstack][:neutron][:file] = "neutron-2014.2.1.tar.gz"
-default[:openstack][:neutron][:target] = "#{node[:openstack][:location]}"
-default[:openstack][:neutron][:installed] = "#{node[:python][:path]}\\Scripts\\neutron-hyperv-agent.exe"
+default[:openstack][:neutron][:binary] = "#{node[:python][:scripts]}\\neutron-hyperv-agent.exe"
+default[:openstack][:neutron][:config] = "#{node[:openstack][:config]}\\neutron"
 
-default[:openstack][:ceilometer][:name] = "ceilometer"
-default[:openstack][:ceilometer][:version] = "2014.2.1"
-default[:openstack][:ceilometer][:file] = "ceilometer-2014.2.1.tar.gz"
-default[:openstack][:ceilometer][:target] = "#{node[:openstack][:location]}"
-default[:openstack][:ceilometer][:installed] = "#{node[:python][:path]}\\Scripts\\ceilometer-agent-compute.exe"
-default[:openstack][:ceilometer][:lock_path] = "C:\\OpenStack\\var\\run\\"
-default[:openstack][:ceilometer][:signing_dir] = "C:\\OpenStack\\var\\cache\\ceilometer\\keystone-signing\\"
+default[:openstack][:networking_hyperv][:name] = "networking-hyperv"
 
-default[:openstack][:instances] = "C:\\OpenStack\\Instances"
-default[:openstack][:config] = "C:\\OpenStack\\etc"
-default[:openstack][:bin] = "C:\\OpenStack\\bin"
-default[:openstack][:log] = "C:\\OpenStack\\log"
+default[:openstack][:ceilometer][:name] = "ceilometer"
+default[:openstack][:ceilometer][:binary] = "#{node[:python][:scripts]}\\ceilometer-agent-compute.exe"
+default[:openstack][:ceilometer][:config] = "#{node[:openstack][:config]}\\ceilometer"
+default[:openstack][:ceilometer][:lock_path] = "#{node[:openstack][:location]}\\var\\run\\"
+default[:openstack][:ceilometer][:signing_dir] = "#{node[:openstack][:location]}\\var\\cache\\ceilometer\\keystone-signing\\"
```

Comment:
Metrics/LineLength: Line is too long. [125/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)